internal/k8shandler/reconciler.go

```diff
@@ -72,6 +73,7 @@ func SecretReconcile(requestCluster *elasticsearchv1.Elasticsearch, requestClien
 			if err := requestClient.Status().Update(context.TODO(), cluster); err != nil {
 				return err
 			}
+			metrics.IncrementRestartCounterCert()
```

Comment:
i think this is going to increment this twice... if we have it here in this reconciler but then also in the code where we perform the cert restart

Reply:
Corrected